### PR TITLE
feat: add edv zcap

### LINKED ISSUE #81: add edv zcap

### PR DESCRIPTION
**Pull request by fqutishat**
closes #81

Signed-off-by: Firas Qutishat <firas.qutishat@securekey.com>

### CODE REVIEW

File: cmd/agent-js-worker/package.json [hunk 1]
```diff
@@ -44,7 +44,7 @@
     "webpack-shell-plugin": "0.5.0"
   },
   "engines": {
-    "node": ">=12.14",
+    "node": ">=12.13",
```
Comment (rolsonquadras) [inline cmd/agent-js-worker/package.json:47]: Version is decreased from 14 tp 13. Is this intentional ?

**Review by rolsonquadras — APPROVED** [review]

File: cmd/agent-js-worker/package.json (hunk 1 above)
Comment (fqutishat) [reply cmd/agent-js-worker/package.json:47]: yes

**Review by soluchok — APPROVED** [review]

File: pkg/auth/zcapld/service.go [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+
+	err = hs.Sign(capability.Invoker, req)
+	if err != nil {
+		return nil, err
+	}
+
+	return &req.Header, nil
+}
+
+func compressZCAP(zcap *zcapld.Capability) (string, error) {
```
Comment (sandrask) [inline pkg/auth/zcapld/service.go:88]: I have seen this function multiple times - should this be part of common code in edge-core? @fqutishat @llorllale

Comment (fqutishat) [reply pkg/auth/zcapld/service.go:88]: yes

Comment (llorllale) [reply pkg/auth/zcapld/service.go:88]: yes

**Pull request merged**